lib/index.mjs

```diff
@@ -0,0 +1,35 @@
+let createRequire = () => {
+  throw new Error(
+    "createRequire is not supported in browser of test environment.",
+  );
+};
+
+try {
+  createRequire = await import("module").then((m) => m.createRequire);
```

Comment:
```suggestion
  createRequire = await import("node:module").then((m) => m.createRequire);
```

We can also add fallback in `.cache`

Reply:
What do you mean by `.cache`?

Reply:
I meaning chaining promise with catch! Will add code refactor.